install/kubernetes/helm/istio/templates/sidecar-injector-configmap.yaml

```diff
@@ -130,7 +130,15 @@ data:
             fieldRef:
               fieldPath: metadata.name
         - name: ISTIO_META_INTERCEPTION_MODE
-          value: {{ "[[ annotation .ObjectMeta `sidecar.istio.io/interceptionMode` .ProxyConfig.InterceptionMode ]]" }}
+          value: {{ "[[ or (index .ObjectMeta.Annotations \"sidecar.istio.io/interceptionMode\") .ProxyConfig.InterceptionMode.String ]]" }}
+        {{ "[[ range $k,$v := .ObjectMeta.Annotations ]]" }}
```

Comment:
FWIW, this causes me issues. In particular:

```
Error creating: Pod "mongodb-v1-787d5d8494-69j2f" is invalid: spec.containers[1].env[5].name: Invalid value: "ISTIO_METAB64_a3ViZXJuZXRlcy5pby9saW1pdC1yYW5nZXI=": a valid environment variable name must consist of alphabetic characters, digits, '_', '-', or '.', and must not start with a digit (e.g. 'my.env-name',  or 'MY_ENV.NAME',  or 'MyEnvName1', regex used for validation is '[-._a-zA-Z][-._a-zA-Z0-9]*')
```

Base64 encodings do not always pass this regex for validation.